checkstyle-suppressions.xml

```diff
@@ -22,4 +22,7 @@ limitations under the License.
   <suppress files=".*\.java" checks="JavadocParagraph" lines="1-12"/>
```

Comment:
we shouldn't do any java checkstyle at all now that we have googlejavaformat ey?

Reply:
Checkstyle does things that google java format doesn't. Such as enforcing copyright header and javadoc requirements.